hack/make-rules/test-integration.sh

```diff
@@ -37,6 +37,8 @@ KUBE_TIMEOUT="-timeout 600s"
 KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=${KUBE_INTEGRATION_TEST_MAX_CONCURRENCY:-"-1"}
 LOG_LEVEL=${LOG_LEVEL:-2}
 KUBE_TEST_ARGS=${KUBE_TEST_ARGS:-}
+# Default glog module settings.
+KUBE_TEST_VMODULE=${KUBE_TEST_VMODULE:-"garbagecollector*=6,graph_builder*=6"}
```

Comment:
Worked fine for me like this:

`make test-integration GOFLAGS="-v" WHAT=./test/integration/garbagecollector KUBE_TEST_VMODULE="garbagecollector=1,graph_builder=1" KUBE_TEST_ARGS="-race --run ^TestMixedRelationships"`